tests/disableHttp/disableHttp.js

```diff
@@ -12,15 +12,24 @@
 'use strict';
 
 const supertest = require('supertest');
-const api = supertest(require('../../express').app);
+const expect = require('chai').expect;
 const constants = require('../../api/v1/constants');
+const tu = require('../testUtils');
 const path = '/v1/users';
-const expect = require('chai').expect;
 
 describe('tests/disableHttp/disableHttp.js, http is disabled >', () => {
```

Comment:
should also test that it isn't rejected when sent with https.

Reply:
yeah i wanted to do that but it ended up being way too much effort, tried to not get bogged down in setting up ssl for tests

at least it doesn't test *less* than it was testing before

Reply:
ok - you verified manually?